docs + blog: image update (#166)
spokestack/spokestack-website: content/docs/Design/tips-for-designing-visual-output.md

- Avoid using a toggle. Many users assume toggles control listening for the wake word, similar to how Alexa works.
- Make sure your button is always in the same place. This way, users will know where to look ([golden rule #1:](https://www.cs.umd.edu/users/ben/goldenrules.html) “strive for consistency”).

> ## Make Sure Users Know Who’s Turn It Is to Talk

Reviewer: ```suggestion
## Make Sure Users Know Whose Turn It Is to Talk
```
